packages/stark-core/src/configuration/entities/application/app-config.entity.ts

```diff
@@ -58,28 +59,37 @@ export class StarkApplicationConfigImpl implements StarkApplicationConfig {
 	@autoserialize
 	public loggingFlushResourceName?: string;
 
+	@ValidateIf(StarkValidationMethodsUtil.validateIfDefined)
```

Comment:
This validation should not be used here because this property has a default value is set in the constructor in any case... so this validation will always pass.